src/main/python/pybuilder/plugins/python/pylint_plugin.py

```diff
@@ -44,4 +45,63 @@ def execute_pylint(project, logger):
     logger.info("Executing pylint on project sources")
 
     command_and_arguments = ["pylint"] + project.get_property("pylint_options")
-    execute_tool_on_modules(project, "pylint", command_and_arguments, True)
+    pylint_output_file_path = execute_tool_on_modules(project, "pylint", command_and_arguments, True)[1]
+
+    with open(pylint_output_file_path, 'r') as f:
+        file_content = f.read().splitlines()
+
+    module_name = ''
+    pylint_score = 0
+    pylint_change = 0
+    errors = 0
+    errors_info = ''
+    warnings = 0
+    warnings_info = ''
+    show_info_messages = project.get_property('pylint_show_info_messages')
+    show_warning_messages = project.get_property('pylint_show_warning_messages')
+    for line in file_content:
```

Comment:
If we parse pylint results I think that more suitable is JSON format `--output-format=json` instead of raw text. It's more resistant to future changes, isn't?

Example, of JSON output
```
[
    {
        "message": "Missing module docstring", 
        "obj": "", 
        "column": 0, 
        "path": "src/main/python/some_lib/model.py", 
        "line": 1, 
        "type": "convention", 
        "symbol": "missing-docstring", 
        "module": "some_lib.model"
    }, 
    {
        "message": "Invalid constant name \"lc\"", 
        "obj": "", 
        "column": 0, 
        "path": "src/main/python/some_lib/model.py", 
        "line": 12, 
        "type": "convention", 
        "symbol": "invalid-name", 
        "module": "some_lib.model"
    }
...
```

Reply:
Unfortunately, JSON report doesn't provide scores. :(